modules/game/src/main/java/clockencoder/VarIntEncoder.java

```diff
@@ -29,11 +29,11 @@ public static void writeUnsigned(int n, BitWriter writer) {
 
     public static int readUnsigned(BitReader reader) {
         int n = reader.readBits(6);
-        if ((n & 0x20) != 0) {
-            n ^= 0x20;
+        if (n > 0x1F) {
```

Comment:
the changes to this file are just cleanup by the way, doesn't affect decoding logic at all